['Traceback (most recent call last):\n  File "/var/folders/x3/l43tt5dd3y19dmn0cq8rhm7r0000gq/T/tmpbf70_xvd/solution_0.py", line 70, in <module>\n    print(Solution().resultArray(nums = [2,1,3,3]))\nTypeError: resultArray() got an unexpected keyword argument \'nums\'\n', 'Traceback (most recent call last):\n  File "/var/folders/x3/l43tt5dd3y19dmn0cq8rhm7r0000gq/T/tmpbf70_xvd/solution_1.py", line 70, in <module>\n    print(Solution().resultArray(nums = [5,14,3,1,2]))\nTypeError: resultArray() got an unexpected keyword argument \'nums\'\n']